sources/packages/backend/apps/api/src/route-controllers/education-program/education-program.controller.service.ts

```diff
@@ -22,11 +22,11 @@ import {
   credentialTypeToDisplay,
   getISODateOnlyString,
   getUserFullName,
-  INSTITUTION_TYPE_BC_PRIVATE,
 } from "../../utilities";
 import { CustomNamedError } from "@sims/utilities";
 import { FormNames } from "../../services/form/constants";
 import { EDUCATION_PROGRAM_NOT_FOUND } from "../../constants";
+import { INSTITUTION_TYPE_BC_PRIVATE } from "@sims/test-utils/constants";
```

Comment:
This is used for the application and belongs to the business context, right?
If it is right it does not belong to "@sims/test-utils".
It can be moved to `@sims/utilities` or `@sims/services` depending on its scope.